apps/server/src/modules/user-login-migration/error/restart-user-login-migration.error.ts

```diff
@@ -0,0 +1,30 @@
+import { BusinessError } from '@shared/common';
+import { HttpStatus } from '@nestjs/common';
+import { ErrorLogMessage, Loggable } from '@src/core/logger';
+
+export class RestartUserLoginMigrationError extends BusinessError implements Loggable {
```

Comment:
StartUserLoginMigrationError and RestartUserLoginMigrationError look like it is the same. I see no reasons to split it in seperat ErrorLogables. It also look like you want to extends BadRequestException instead of BusinessError.

Reply:
I introduced an UserLoginMigrationLoggableException instead of the two errors.